akka-http/src/main/scala/akka/http/scaladsl/server/directives/FormFieldDirectives.scala

```diff
@@ -40,6 +59,50 @@ trait FormFieldDirectives extends ToNameReceptacleEnhancements {
 }
 
 object FormFieldDirectives extends FormFieldDirectives {
+
+  val _formFieldSeq: Directive1[immutable.Seq[(String, String)]] = {
+    import BasicDirectives._
+    import FutureDirectives._
+    import akka.http.scaladsl.unmarshalling._
+
+    extract { ctx ⇒
+      import ctx.{ executionContext, materializer }
+      Unmarshal(ctx.request.entity).to[StrictForm].fast.flatMap { form ⇒
+        val fields = form.fields.collect {
+          case (name, field) if name.nonEmpty ⇒
+            Unmarshal(field).to[String].map(fieldString ⇒ (name, fieldString))
```

Comment:
Yes, let's do that. I made it this ticket: https://github.com/akka/akka/issues/19080